src/App.js

```diff
@@ -1,9 +1,23 @@
 import "./App.css";
+import { MainContainer } from "./shared/layout/MainContainer";
 
 function App() {
   return (
     <div className="text-center text-2xl justify-center items-center h-screen">
-      Hello World
+      <MainContainer>
+        <h1>Main Container</h1>
```

Comment:
Same here. Use <Typography variant=‘h1’></> instead. It’s not actually important for this changes but you can keep this in your mind.